hack/lib/etcd.sh

```diff
@@ -77,6 +77,10 @@ kube::etcd::start() {
 
   # Start etcd
   ETCD_DIR=${ETCD_DIR:-$(mktemp -d 2>/dev/null || mktemp -d -t test-etcd.XXXXXX)}
+  # Using a ramdisk to get better performance
```

Comment:
please add `if` conditions to limit this to arm platform, so folks looking at this script will realize that this is a platform specific thing not needed for other platforms.